project-service/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "turtle",
+  "version": "1.0.0",
+  "description": "Project service for the fastest agile project management app",
+  "main": "index.js",
+  "scripts": {
+    "start": "babel-node ./src/index.js",
+    "test": "gulp project-service:test"
+  },
+  "repository": {
```

Comment:
Since we are sharing the repository with the other services, I think we should remove all repository related data from the child package.jsons